cassandra/src/main/scala/filodb.cassandra/metastore/CheckpointTable.scala

```diff
@@ -36,7 +36,7 @@ sealed class CheckpointTable(val config: Config,
       s"""SELECT groupnum, offset FROM $tableString WHERE
          | databasename = ? AND
          | datasetname = ? AND
-         | shardnum = ? """.stripMargin).setConsistencyLevel(ConsistencyLevel.QUORUM)
+         | shardnum = ? """.stripMargin).setConsistencyLevel(ConsistencyLevel.ONE)
```

Comment:
@shaldank thanks for the PR.. please change it to `LOCAL_QUORUM`

Reply:
Done.